### DISCUSSION TITLE
admin panel (username and password)

### DISCUSSION THREAD

**Discussion by 3bodymo** [JasperAlexander/seaport-backend#1] [General]
At first, thank you for this awesome project.

I want to write a note here because this problem took me a while to find the solution because I don't know a lot about Django framework and it would help someone else. For admin panel you have to create your own username and password to get access to it.
Just run the following command in your terminal:

```shell
python manage.py createsuperuser
```

Then enter username, email, and password.

**Comment by JasperAlexander** [comment]
Thank you for this note! I will update the code within the coming months.
